open-api/rest-catalog-open-api.yaml

```diff
@@ -3789,6 +4097,21 @@ components:
             EmptyResponse:
               $ref: '#/components/examples/ListNamespacesEmptyExample'
 
+    PayLoadTooLargeResponse:
+      description:
+        Payload is too large, clients should leverage the preplan endpoint and provide a plan-task before doing a plan.
```

Comment:
This description should be generic. The planTable specific descriptions should be put to that endpoint description

Reply:
Will make description generic.